regression/goto-instrument/fp-reachability-slice1/test.desc

```diff
@@ -0,0 +1,8 @@
+CORE
+main.c
+--fp-reachability-slice c
+^EXIT=0$
+^SIGNAL=0$
+--
```

Comment:
This is a bit easy to pass at the moment, especially if the precise pretty-printing of "dead f" were to change. Suggest adding some positive requirements too, to make the test more robust.